<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusViewRubbishBoat3</class>
 <widget class="QWidget" name="StatusViewRubbishBoat3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1013</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="statusWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>620</y>
     <width>1891</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 16, 48);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="widget_6" native="true">
      <widget class="SpeedWidget2" name="widget_7" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>371</width>
         <height>341</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="AttitudeWidget" name="widget_8" native="true"/>
    </item>
    <item>
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="AltitudeWidget" name="widget_9" native="true">
      <widget class="QSlider" name="verticalSlider">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>49</y>
         <width>20</width>
         <height>241</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>300</y>
         <width>331</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget_10" native="true">
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(211, 215, 207);</string>
       </property>
       <property name="text">
        <string>参数信息：</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>151</width>
         <height>112</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>航向角：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="yaw_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>加速度_x：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="ax_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>加速度_y：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="ay_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>加速度_z：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="az_label">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>000</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>221</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonSerial">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>连接</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_9">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>参数调整</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_10">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>执行任务</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_11">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>数据记录</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_12">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>紧急恢复</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_13">
          <property name="styleSheet">
           <string notr="true">color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>断开连接</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>220</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>81</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
   border: 2px solid white;
   border-radius: 5px;
   background-color: #02112E;
   color:white
}
QProgressBar {
   border: 2px solid white;
   border-radius: 5px;
   text-align: right;
}</string>
       </property>
       <property name="value">
        <number>60</number>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::BottomToTop</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(211, 215, 207);</string>
       </property>
       <property name="text">
        <string>电量：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonModeSwitchM">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>270</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 236);</string>
       </property>
       <property name="text">
        <string>手动</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonModeSwitchH">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>270</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 236);</string>
       </property>
       <property name="text">
        <string>制动</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonModeSwitchD">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>270</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 236);</string>
       </property>
       <property name="text">
        <string>自动</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(211, 215, 207);</string>
       </property>
       <property name="text">
        <string>模式：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonClearSwitch">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>270</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 236);</string>
       </property>
       <property name="text">
        <string>回收装置控制</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="videoWidget" native="true">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>0</y>
     <width>741</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="VlcWidgetVideo" name="videoPlayWidget" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="videoControlWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="videoAddEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="playButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(115, 210, 22);</string>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="styleSheet">
          <string notr="true">color: rgb(239, 41, 41);</string>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWebEngineView" name="mapWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>613</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AttitudeWidget</class>
   <extends>QWidget</extends>
   <header>attitudewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AltitudeWidget</class>
   <extends>QWidget</extends>
   <header>altitudewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpeedWidget2</class>
   <extends>QWidget</extends>
   <header>speedwidget2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VlcWidgetVideo</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetVideo.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
